src/components/NavBar.js

```diff
@@ -12,6 +12,7 @@ const NavBar = () => (
       </p>
       <Link to='/' className="App-link"> Home </Link>
       <Link to='/sample' className="App-link"> sample </Link>
```

Comment:
is there a CSS style for`App-link`? if yes you can use CSS in JS

Reply:
This is the sample navbar component. The main navbar component is still being implemented by uche, together with the landing page and footer.

Reply:
Okay cool.